os/board/rtl8730e/src/component/soc/amebad2/fwlib/rom_common/ameba_assert_rom.c

```diff
@@ -11,7 +11,7 @@
 __weak HAL_ROM_TEXT_SECTION
 void io_assert_failed(uint8_t *file, uint32_t line)
 {
-	DBG_8195A("io driver parameters error! file_name: %s, line: %d", file, line);
```

Comment:
you removed ameba_assert_rom.c in the build.

this change is okay for you?

Reply:
Yes~ As the io_assert_failed is linked from assert_param previously. Since now we linked assert_param to Tizen Lite assertion. Then we can remove this file from build now~